spec/components/list_spec.rb

```diff
@@ -0,0 +1,73 @@
+require "rails_helper"
+
+describe "List", type: :view do
+  def component_name
+    "list"
+  end
+
+  it "does not render anything if no data is passed" do
+    test_data = {}
```

Comment:
Does it matter that this test passes an object instead of an array, which the component expects? It still passes, but is it a fair test? Or do we need both tests?

Reply:
The component expects `{ items: ["one", "two", "three"] }` as a minimum - so a test to test for graceful failure with an empty `items` array is a good idea.